dashboard/client/src/App.css

```diff
@@ -0,0 +1,167 @@
+body {
```

Comment:
What are the main things we're overriding in this file?

Reply:
we are overwriting some small amount of bootstrap styles and all other classnames are utility ones (you can use it when you need, if there is no existing one from bootstrap)